push_notification/subscriber/services/subscriber_service.py

```diff
@@ -0,0 +1,48 @@
+from subscriber.models.subscriber import Subscriber
+
+
+class SubscriberService(object):
+
+    def __init__(self, subscriber, serializer):
+        self.subscriber = subscriber
+        self.serializer = serializer
+
+    def get_subscriber(self, pk=None):
+        "Return subscriber details by given id"
+        if pk is not None:
+            queryset = self.subscriber.objects.get(id=pk)
+            response = self.serializer(queryset, many=False)
+            return response.data
+        return None
+
+    def get_all_subscriber(self):
+        """Return all subscriber registered"""
+        queryset = self.subscriber.objects.all()
+        response = self.serializer(queryset, many=True)
+        return response.data
+
+    def add_subscriber(self, data):
+        """Register subscriber with given data"""
+        subscriber = self.serializer(data=data)
+        if subscriber.is_valid(raise_exception=True):
+            subscriber.save()
```

Comment:
Lets use the normal Serializer instead of ModelSerializer 
So we can use the serializer for validation only and save the validated_data to create the object using ORM.